iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Onboarding/BaseOnboardingViewController.swift

```diff
@@ -30,7 +30,13 @@ class BaseOnboardingViewController: PresenterViewController<OnboardingSubView>,
         buttonBackground.backgroundColor = UIColor.clear
         buttonBackground.addSubview(button)
         button.addTarget(self, action: #selector(primayTapped), for: .touchUpInside)
-        let gradientBackgroundHeight: CGFloat = 100.0
+
+        var gradientBackgroundHeight: CGFloat = 100.0
+        if #available(iOS 11, *),
+           let bottomMargin = UIApplication.shared.windows.filter({ $0.isKeyWindow }).first?.safeAreaInsets.bottom,
+           bottomMargin > 0 {
+            gradientBackgroundHeight += bottomMargin
+        }
```

Comment:
This check is obsolete, we only support iOS >= 13 atm

Reply:
So on iOS the "deployment target" is equivalent to "minimum target"? Thanks for the information, did not know this 👍